cli/cmd/root.go

```diff
@@ -150,21 +175,63 @@ func init() {
 	// Cobra supports persistent flags, which, if defined here,
 	// will be global for your application.
 	rootCmd.PersistentFlags().StringVar(&cfgFile, "config", "", "config file (default is $HOME/.vmclarity.yaml)")
-	rootCmd.PersistentFlags().StringVar(&output, "output", "", "set output directory path. Stdout is used if not set.")
+	rootCmd.PersistentFlags().StringVar(&output, "output", "", "set output directory path. Stdout is used if not set")
 	rootCmd.PersistentFlags().StringVar(&server, "server", "", "VMClarity server to export scan results to, for example: http://localhost:9999/api")
+	rootCmd.PersistentFlags().StringVar(&scanConfigName, "scan-config-name", "", "use an existing scan config that is defined in the VMClarity server")
```

Comment:
I'm suggesting that we support both, some people might prefer to use the ID in case someone on the server side renames the config and breaks everything